README.md

```diff
@@ -1,109 +1,105 @@
+```plain
+mongodb+srv://<username>:<password>@cluster0.7k5er.mongodb.net/<database_name>
+```
 
-1. Type `mongo --help` to check MongoDB is on the system path. If you see the help text displayed, skip to the next prerequisite.
-2. Navigate to your MongoDB installation folder and copy the full path to the bin folder (default should be `C:\Program Files\MongoDB\Server\4.4\bin`)
-3. Add the MongoDB bin folder to the system path
-   1. Type 'environment' in the Windows Search box and open "Edit the system environment variables" (or right-click "This PC" > Properties > Advanced System Properties)
-   2. Click "Environment Variables..."
-   3. Select "Path" under "System variables" and click "Edit"
-   4. Click "New"
-   5. Paste the path you copied above, or click "Browse..." to locate the folder on your system
-   6. Click "OK"
-   7. Click "OK"
-   8. Click "OK"
-4. Ensure MongoDB service is running
-   1. Type 'Services' in the Windows Search box and open "Services".
-   2. Scroll down the list and ensure "MongoDB" is running. Click to select the service and click the "play" icon to start if not.
-#### MongoDB Database Tools installed
+For the example app the database name is `example_db`.
 
-1. Check `mongoimport` is installed by running `mongoimport --help` in your command prompt. If you see the help text displayed, continue to the next prerequisite. If you see a `command not recognized` error message, continue to next step.
-2. Follow the installation instructions for [macOS](https://docs.mongodb.com/database-tools/installation/installation-macos/#installation) or [Windows](https://docs.mongodb.com/database-tools/installation/installation-windows/#installation)
+You'll also see the `PORT` for your API in this file. Do not change this `PORT` number.
 
-### Fork Project
+🛑 **YOUR ENVIRONMENT VARIABLES SHOULD NEVER BE COMMITED AND THE `.env` FILE HAS ALREADY BEEN ADDED TO THE [`.gitignore`](./.gitignore).** 🛑
 
-1. Click on `Fork`
-2. Select your Username
+### Populating The Database
 
-You can also refer to the [GitHub Fork Tutorial](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo)
+If you choose to populate your database with some initial data you can do so using seed data. We have provided an example of seed data in the [`data.example/`](./server/data.example) folder in a file called [`profiles.mongodb`](server/data.example/profiles.mongodb).
 
-This will add the `black-codher-personal-project` to your GitHub account.
+All you need to do is rename the `data.example` folder to `data` and then create a `.mongodb` file that will create and insert data into your database and the correct collection(s).
 
-### Clone Repo
+To get the example app up and running you need to create an `example_db` database, a `profiles` collection and inserting some profile data. This can all be done by running the [`profiles.mongodb`](server/data.example/profiles.mongodb) file in your MongoDB playground.
```

Comment:
Should be "To get the example app up and running, you need to create an `example_db` database, a `profiles` collection and insert some profile data."?